packages/mesos/extra/make_isolators.py

```diff
@@ -0,0 +1,71 @@
+#!/opt/mesosphere/bin/python3
+
+import os
+import subprocess
+import sys
+
+class IsolatorDiscoveryException(Exception):
+        pass
+
+def has_nvidia():
+    """
+    Test if we have nvidia available
+    """
+    nvidia_smi_binary = '/bin/nvidia-smi'
+
+    # Make sure file is there and with correct permissions
+    if not os.path.isfile(nvidia_smi_binary):
+        return False
+    if not os.access(nvidia_smi_binary, os.X_OK):
+        return False
+
+    # Try to run it
```

Comment:
[readability review] - please kill all the comments in this file. In practice, I have seen comments that explain "why" (when needed) are helpful.  Comments that explain "what" should be avoided.